tests/test_view.py

```diff
@@ -208,3 +208,25 @@ def test_cache_throws_exception_with_incorrect_cache_type(self):
             view(
                 'test_exception', {'test': 'test'}
             ).cache_for(1, 'monthss')
+
+    def test_can_add_tests_to_view(self):
```

Comment:
This test I think does too much. Maybe could be separated in smaller tests

def test_that_admin_user_renders_true()
def test_that_normal_user_renders_false()

Reply:
you're right. I sort of snuck it in at the last minute to see if those assertions actually pass. I can refactor as we go though. I'll just need to make a mental note there